updating/index.adoc

```diff
@@ -66,12 +66,12 @@ xref:../updating/updating-cluster-rhel-compute.adoc#updating-cluster-rhel-comput
 * xref:../updating/updating-cluster-rhel-compute.adoc#rhel-compute-updating-minor_updating-cluster-rhel-compute[Updating {op-system-base} compute machines in your cluster]
 
 [id="updating-clusters-overview-update-restricted-network-cluster"]
-== Updating a disconnected cluster
-xref:../updating/updating-restricted-network-cluster/index.adoc#about-restricted-network-updates[Updating a disconnected cluster]: If your mirror host cannot access both the internet and the cluster, you can mirror the images to a file system that is disconnected from that environment. You can then bring that host or removable media across that gap. If the local container registry and the cluster are connected to the mirror host of a registry, you can directly push the release images to the local registry.
+== Updating a disconnected environment
```

Comment:
I think you need to still say that you're updating a cluster. I'd recommend something like "Updating a cluster in a disconnected environment".

Reply:
I'll change the doc section title and all references to it to your suggestion, thanks.